dask/base.py

```diff
@@ -593,6 +593,8 @@ def visualize(*args, **kwargs):
     color : {None, 'order'}, optional
         Options to color nodes.  Provide ``cmap=`` keyword for additional
```

Comment:
If we're being thorough, then `cmap` should be a separate kwargs in this list with a better description than given here; or just remove it, since we don't say anywhere what kind of object cmap can be (and it is probably not tested).

Reply:
Right!